cnd/Cargo.toml

```diff
@@ -1,41 +1,41 @@
 [package]
-authors = ["CoBloX <hello@coblox.tech>"]
 name = "cnd"
```

Comment:
Let's remove cargo toml from the CI/makefile then.

Reply:
So actually, our manifest files are not formatted correctly given the guidelines that are recommended here: https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/cargo.md?rgh-link-date=2020-11-10T07%3A37%3A52Z

See also https://github.com/dprint/dprint/issues/321 for more discussion!

Reply:
What I mean is that cargo tomlfmt is still mentioned in the makefile, I think it should be removed.